ios/FluentUI/Badge Field/BadgeField.swift

```diff
@@ -786,6 +795,11 @@ open class BadgeField: UIView {
         originalDeviceOrientation = UIDevice.current.orientation
     }
 
+    @objc private func handleContentSizeCategoryDidChange() {
+        invalidateIntrinsicContentSize()
```

Comment:
should the order swap so you calculate the height first before triggering invalidation?

Reply:
Shouldn't matter. `invalidateIntrinsicContentSize` sets a flag on the view that will require a re-layout on the next layout pass (effectively the next iteration of the run loop), but doesn't directly trigger a synchronous layout. And all this is done on the main thread, so there's no race condition to worry about.